Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerCustomFolderViewModel.cs

```diff
@@ -44,7 +44,7 @@ public sealed class CodeExplorerCustomFolderViewModel : CodeExplorerItemViewMode
 
         public string FullPath { get; }
 
-        public string FolderAttribute => $"@Folder(\"{FullPath.Replace("\"", string.Empty)}\")";
+        public string FolderAttribute => $"'@Folder(\"{FullPath.Replace("\"", string.Empty)}\")";
```

Comment:
Where is this used? I am a bit concerned about this change in connection with annotation lists.

Reply:
This is only used for display at the bottom of the TreeView when the folder is selected. I was thinking that it made more sense to add the comment marker to make it easier to copy and paste into an existing document.